### PR TITLE
Add new blockchains

### PR DESCRIPTION
**Pull request by Cryptocool1**

### CODE REVIEW

File: blockchains/fusion/info/info.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+    "name": "Fusion",
+    "type": "coin",
+    "symbol": "FSN",
+    "decimals": 18,
+    "website": "https://fusion.org/",
+    "description": "Fusion Protocol is a infrastructure of digital finance. It developed DCRM cross chain technology, time framing technology and Quantum Swap technology to provide an infrastructure of transferring value across different chains, system and time.",
+    "explorer": "https://blocks.fusionnetwork.io/#!/address/0x2b8bb627ce7c0c9cbe579e83099de665032d08be",
```
Comment (vcoolish) [inline blockchains/fusion/info/info.json:8]: https://blocks.fusionnetwork.io/#!

Comment (Inikowu) [reply blockchains/fusion/info/info.json:8]: 539u

Comment (Inikowu) [reply blockchains/fusion/info/info.json:8]: 5390

File: blockchains/fusion/info/info.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
+{
+    "name": "Fusion",
+    "type": "coin",
+    "symbol": "FSN",
+    "decimals": 18,
+    "website": "https://fusion.org/",
+    "description": "Fusion Protocol is a infrastructure of digital finance. It developed DCRM cross chain technology, time framing technology and Quantum Swap technology to provide an infrastructure of transferring value across different chains, system and time.",
+    "explorer": "https://blocks.fusionnetwork.io/#!/dashboard",
```
Comment (vcoolish) [inline blockchains/fusion/info/info.json:8]: "https://blocks.fusionnetwork.io/#!"

File: blockchains/ethereum/chainlist.json [hunk 3]
```diff
@@ -27,6 +27,19 @@
...
+      },
+      {
+         "chain": "42262",
+         "coinId": "oasis"
+      },
+      {
+         "chain": "7700",
+         "coinId": "canto"
       }
+
```
Comment (vcoolish) [inline blockchains/ethereum/chainlist.json:43]: add others here too

File: blockchains/ethereum/chainlist.json [hunk 4]
```diff
@@ -27,6 +27,78 @@
...
+         "chain": "128",
+         "coinId": "heco"
+      },
+      {
+         "chain": "1285",
+         "coinId": "moonriver"
+      },
+      {
+         "chain": "2222",
+         "coinId": "kava"
```
Comment (vcoolish) [inline blockchains/ethereum/chainlist.json:81]: this is different chain, you can create other folder for it like kavaevm

File: blockchains/ethereum/chainlist.json [hunk 5]
```diff
@@ -27,6 +27,78 @@
...
+         "chain": "321",
+         "coinId": "kucoin"
+      },
+      {
+         "chain": "70",
+         "coinId": "hoo"
+      },
+      {
+         "chain": "70",
+         "coinId": "hoo"
```
Comment (vcoolish) [inline blockchains/ethereum/chainlist.json:61]: duplicate

File: blockchains/okexchain/info/info.json [hunk 6]
```diff
@@ -0,0 +1,27 @@
+{
+    "name": "Elk Finance",
```
Comment (vcoolish) [inline blockchains/okexchain/info/info.json:2]: why you have elk in okex folder?

**Review by vcoolish — APPROVED** [review]

**Comment by vcoolish** [comment]
@cryptomanz1 please add [rcp_url](https://chainlist.org/) (string) and [slip44](https://github.com/satoshilabs/slips/blob/master/slip-0044.md) (int) fields to all ethlike chains

**Pull request merged**